### PR TITLE
HW#8_complete

### PR DESCRIPTION
**Pull request by denzelz**

### CODE REVIEW

File: flask_lesson_intro/main.py [hunk 1]
```diff
@@ -9,5 +10,19 @@ def get_home_page():
...
+def get_author_page():
+    return render_template("author.html")
+
+
+@app.route('/<item>')
+def get_items_page(item):
+    data = get_data()
+    for elem in data:
+        if item == elem['title']:
+            return render_template("items.html", title=elem['title'], text=elem['text'],
```
Comment (dimigor) [inline flask_lesson_intro/main.py:24]: what if yoou do not find satisfied element? Take a look at `for..else`

File: flask_lesson_intro/templates/author.html [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+<p></p>
+<p></p>
+<ul class="navbar-nav mr-auto">
+      <li class="nav-item active">
+        <a class="nav-link" href="https://github.com/denzelz">My GitHub
+          <span class="sr-only">(current)</span>
+        </a>
+      </li>
+    </ul>
+{% endblock %}
```
Comment (dimigor) [inline flask_lesson_intro/templates/author.html]: endline

File: flask_lesson_intro/templates/items.html [hunk 3]
```diff
@@ -0,0 +1,8 @@
+{% extends 'base.html' %}
+{% block content%}
+<h1>{{ title }}</h1>
+    <img src="{{url_for('static', filename=item_img)}}" usemap="#image-map" width="360px" height="400px">
+    <p class="text">{{ text }}</p>
+    <p></p>
+    <p>Words counter: {{ count }}</p>
+{% endblock %}
```
Comment (dimigor) [inline flask_lesson_intro/templates/items.html]: endline

**Review by dimigor — CHANGES_REQUESTED** [review]
Hmmm do not see some files in this pull request

File: flask_lesson_intro/main.py (hunk 1 above)
Comment (denzelz) [reply flask_lesson_intro/main.py:24]: added exception handling, thanks

File: flask_lesson_intro/templates/author.html (hunk 2 above)
Comment (denzelz) [reply flask_lesson_intro/templates/author.html]: fix

File: flask_lesson_intro/templates/items.html (hunk 3 above)
Comment (denzelz) [reply flask_lesson_intro/templates/items.html]: fix

File: flask_lesson_intro/main.py [hunk 4]
```diff
@@ -9,5 +11,26 @@ def get_home_page():
...
+                                   item_img=item + '.png', count=len(elem['text'].split()))
+    else:
+        raise BadRequest
```
Comment (dimigor) [inline flask_lesson_intro/main.py:27]: Better will be render template with 404 error

**Review by dimigor — APPROVED** [review]

**Pull request merged**